core-bundle/src/Security/TwoFactor/Authenticator.php

```diff
@@ -55,7 +55,7 @@ public function getProvisionUri(User $user, Request $request): string
     public function getQrCode(User $user, Request $request): string
     {
         $renderer = new ImageRenderer(
-            new RendererStyle(180, 0),
+            new RendererStyle(180, 1),
```

Comment:
```suggestion
            new RendererStyle(180, 3),
```

In #6780 we introduced a 3px border.

May be this should be configurable though?

Reply:
```php
public function __construct(
        int $size,
        int $margin = 4,
        ?ModuleInterface $module = null,
        ?EyeInterface $eye = null,
        ?Fill $fill = null
    ) {
```

Having this set to 0 is already a bug to me, however, the margin is way bigger with 3 (I did test it out really quick)

Reply:
1:
![image](https://github.com/user-attachments/assets/b316e332-631a-4e88-a106-488e853cef36)

3:
![image](https://github.com/user-attachments/assets/cf96fc0c-ec95-4ca0-ae19-d0b0343c50c2)

Default:
![image](https://github.com/user-attachments/assets/fb51c558-dd2f-46e5-b951-b952b53b567f)

Reply:
> May be this should be configurable though?

Depends on what you would want to choose but if people want a bigger border in the frontend, they'd style it 🤔?
Maybe just winging with the default from the library itself would be better?

Reply:
Ah, yes never mind, forgot that the margin setting is not related to pixels.

Reply:
I think we are fine with 3px for everyone. Android need this still.

Reply:
![grafik](https://github.com/user-attachments/assets/7d8cb129-353b-454a-826c-ed2856c19542)

Reply:
Sorry i dint see the whole conversation. Better for configuration is to make this code bigger. The first parameter.

Reply:
The margin is related to one dot.

Reply:
I'am fine with `null`(4), `1` or `3` as default.